satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -386,27 +396,119 @@ def _get_reader_with_filehandlers(filenames, reader_configs):
     clear_cache(reader)
     return reader
 
-
 def clear_cache(reader):
     """Clear the cache for file handlres in reader."""
     for key in reader.file_handlers:
         fhs = reader.file_handlers[key]
         for fh in fhs:
             fh.cached_file_content = {}
 
+list_channel_solar = ["vis_04", "vis_05", "vis_06", "vis_08", "vis_09",
```

Comment:
question and comment... for consistency and maintainability should these variables here be listed as constants at the beginning of the file?